tests/director/tests/application_api_test.go

```diff
@@ -1580,6 +1580,16 @@ func TestMergeApplications(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, newFormation, formation.Name)
 
+	defer func() {
+		t.Logf("Cleaning up formation: %s", newFormation)
+
+		var response graphql.Formation
+		deleteFormationReq := fixtures.FixDeleteFormationRequest(newFormation)
+		err = testctx.Tc.RunOperation(ctx, certSecuredGraphQLClient, deleteFormationReq, &response)
+
+		t.Logf("Deleted formation: %v", response)
```

Comment:
You could print only the formation name instead of the whole object 
```suggestion
		t.Logf("Deleted formation with name: %s", response.Name)
```

Reply:
done